test/core/relaxed-simd/relaxed_dot_product.wast

```diff
@@ -62,6 +65,7 @@
                        (v128.const i32x4 1 2 3 4))
                (either
                  (v128.const i32x4 -66047 2 3 4)
+                 (v128.const i32x4 -65536 2 3 4)
```

Comment:
Oh running this locally I think this needs to be `-65535` to account for the `+1`